### DISCUSSION TITLE
Ring in mobile device

### DISCUSSION THREAD

**Discussion by aksld** [tailwindlabs/tailwindcss#5882] [Help]
Hello,

I have problem with focus:ring in mobile device.

I made a codepen : https://codepen.io/aksld/pen/OJjmyLJ

On my iPhone, when clicking on input, there is no ring/border.

Is this a known issue ?

Thanks for help !

**Comment by adamwathan** [comment]
Hey! Seems like an iOS Safari-specific thing for how inputs are styled. Adding `appearance-none` to the input to remove the browser specific styling fixes it in my testing.

**Reply by lakkes-ra** [reply]
That one did the trick.
